### DISCUSSION TITLE
FluentDialog - can not force it to enable scrolling when content of FluentDialogBody dynamically extends

### DISCUSSION THREAD

**Discussion by ZnowuJa** [microsoft/fluentui-blazor#2596] [Q&A] [answered]
Hi,

In my current project I meet with problem that I cannot force FluentDialog to scroll FluentDialogBody when content of it dynamically grows. 
In this particular example in my FluentDialog I have a Tabs component with Roles in Organisation. In each tab there is a component of RoleList which additionally contain another component of RoleEditor. RoleList component can have many RoleEditors and all this content may grow rapidly, so some controls are outside of not only FluentDialogBody but also FluentDialog.

This is how I define dialog and set PreventScroll to false:
`        var dialog = await _dialogService.ShowDialogAsync<OrganisationEditComponent>(item, new DialogParameters()
            {
                Height = "750px",
                Width = "800px",
                Title = item.Id > 0 ? "Update Organisation..." : "Create new Organisation...",
                PreventDismissOnOverlayClick = true,
                PreventScroll = false
            });`

This is a picture what happens and how  controls are rendered outside of FluentDialog:

<img width="474" alt="image" src="https://github.com/user-attachments/assets/6d5912c8-39f6-4e3f-be27-0388f397ebc0">

Folder in repo with code of page and components: [folder](https://github.com/ZnowuJa/BLazorFluentTestApp/tree/master/BOAppFluentUI/Components/Pages/Organisations). 
This is not working project, just repo with files of page and components.

Is there something I do wrong? Or maybe it is not the way to do this, and I should try another attempt (like do it on page not in Dialog)?

**Comment by vnbaaij** [comment, marked as answer]
You need to do something with the `Style` of the `FluentDialogBody`. like adding `overflow: scroll;`:

![image](https://github.com/user-attachments/assets/971e596b-a582-4eb9-b1bc-6b4f957c035c)

`PreventScroll` is to stop scrolling the content behind the dialog,

**Comment by ZnowuJa** [comment]
Thanks.
I finished with: 
`<FluentDialogBody  Style="overflow-y: auto; overflow-x: hidden;">`
And it works perfect!
